index.html

```diff
@@ -645,7 +645,34 @@ <h2>A project for first-time contributions</h2>
             </div>
         </div>
         <!-- ________ *CodesWithRobi* Contributor card END ________  -->
-
+        
+        <!-- Arin Mandal Contributor card START -->
+        <div class="card">
+            <p class="name">Arin Mandal</p>
+            <p class="contact">
+                <i class="fab fa-twitter"></i>
+                <a href="https://www.twitter.com/" target="_blank">@ArinMandal</a>
```

Comment:
The twitter link should be redirected to your twitter profile.
```suggestion
                <a href="https://www.twitter.com/ArinMandal" target="_blank">@ArinMandal</a>
```